web/supabase/config.toml

```diff
@@ -19,7 +19,8 @@ max_rows = 1000
 port = 54322
 # The database major version to use. This has to be the same as your remote database's. Run `SHOW
 # server_version;` on the remote database to check.
-major_version = 14
+shadow_port = 54320
```

Comment:
Can we not change the default shadow port?

Reply:
Actually this is the default shadow port. 

https://supabase.com/docs/reference/cli/config

Reply:
I ended up removing this, because it was failing to push with this on with my testing. Not sure why